<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <author>Evan Teran</author>
 <class>DialogInputValue</class>
 <widget class="QDialog" name="DialogInputValue" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Input Value</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Hexdecimal</string>
     </property>
     <property name="buddy" >
      <cstring>hexInput</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="hexInput" />
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Signed</string>
     </property>
     <property name="buddy" >
      <cstring>signedInput</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="signedInput" />
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Unsigned</string>
     </property>
     <property name="buddy" >
      <cstring>unsignedInput</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="unsignedInput" />
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hexInput</tabstop>
  <tabstop>signedInput</tabstop>
  <tabstop>unsignedInput</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogInputValue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>95</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel" >
     <x>55</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogInputValue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>168</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
